### ISSUE TITLE
Increase space between top corner and graph maximum

### ISSUE THREAD

**Issue by LordHepipud** [Icinga/icingaweb2-module-graphite#153]
Some graphs are written into or above the top corner of the graph, making it hard to understand if the graph ih exceeding the top corner or if it is simply drawn on it,

## Expected Behavior

There should be a 10% top margin between the top-graph value and the top frame line.

## Current Behavior

Graphs are written directly on the top corner frame or above it

## Possible Solution

Add 10% top margin for top graph value

## Steps to Reproduce (for bugs)

Unknown - happens to certain graphs

## Your Environment

* Module version (System - About): 1.0.0
* Icinga Web 2 version and modules (System - About): 2.5.1
* Icinga 2 version (`icinga2 --version`): 2.8.2
* Operating System and version: Debian Jessie
* Webserver, PHP versions:

**Comment by LordHepipud** [comment]
![icingaweb2_graphite_graph_bug](https://user-images.githubusercontent.com/5050288/37897718-27cb2d74-30e7-11e8-839c-792ae40a796c.png)

**Comment by Al2Klimov** [comment]
Hello @LordHepipud and thank you for reporting!

If I don't misunderstand you, you mean e.g. Ambient_Temp is hard to determine while being exactly 22 because the line is somewhat cut off (and only the area is shown).
We'll have a look at this.

Best,
AK

**Comment by Al2Klimov** [comment]
@bobapple What's your opinion about this?

**Comment by bobapple** [comment]
@LordHepipud what graphite web version are you using?
I would expect graphite web to add some space automatically, at least in my setup it does. We cannot set a fixed maximum value, since we don't know anything about the values in Icinga Web 2. The only thing that we could do is setting the `ymax` parameter to `auto`. Maybe it helps

**Comment by Al2Klimov** [comment]
@LordHepipud Please could you test this by temporarily adding this line:

```
ymax = "auto"
```

... just after:

```
[default-service.urlparams]
```

in `.../modules/graphite/templates/default.ini`.

**Comment by Al2Klimov** [comment]
Hello again @LordHepipud!

I've scanned all our templates and we never cut our graphs off at the top of the Y axis. Even [disk](https://github.com/Icinga/icingaweb2-module-graphite/blob/3a98d2c47354b76f246225593c0a1c52caedbdfd/templates/disk.ini#L6) just displays the maximum as an additional curve.

## TL;DR

Graphs never exceed the top corner – everything is fine.

Best,
AK

PS: @bobapple What's your opinion about this?

**Comment by return2alfred** [comment]
@LordHepipud Hi,giant,I must say so,could you tell me or us how could I make the change of graph background color just the same like yours.Cause this new version didn't mention it.Really appricate your help me for building a better icingaweb ui.Thanks.
